wiki/models/article.py

```diff
@@ -229,8 +229,8 @@ class BaseRevisionMixin(models.Model):
     # NOTE! The semantics of these fields are not related to the revision itself
     # but the actual related object. If the latest revision says "deleted=True" then
     # the related object should be regarded as deleted.
-    deleted = models.BooleanField(verbose_name=_(u'deleted'))
-    locked  = models.BooleanField(verbose_name=_(u'locked'))
+    deleted = models.BooleanField(verbose_name=_(u'deleted'), default=False)
+    locked  = models.BooleanField(verbose_name=_(u'locked'), default=False)
```

Comment:
We don't need a migration for this since till Django 1.6, the default for `BooleanField` was False.

https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.BooleanField